src/tests/stress/savina/10.ConcurrentSortedLinkedList/Main.enc

```diff
@@ -0,0 +1,171 @@
+-- https://github.com/shamsimam/savina/blob/master/src/main/java/edu/rice/habanero/benchmarks/concsll/SortedLinkedList.java
+-- https://github.com/shamsimam/savina/blob/master/src/main/scala/edu/rice/habanero/benchmarks/concsll/SortedListAkkaActorBenchmark.scala
+-- Sequential sorted linked list ported from the first file, then it's wrapped
+-- inside an actor just like how it's done in the second file
+
+import Random
+
+read trait Ord
+  require def compare(o:Ord) : int
+end
+
+read class Integer : Ord(i)
+  val i:int
+
+  def init(i:int) : unit
+    this.i = i
+  end
+
+  def compare(other:Ord) : int
+    val y = EMBED (Integer) (void*)#{other}; END
+    this.i - y.i
+  end
+end
+
+local class Node : Id
```

Comment:
Could this class be made `subord` rather than `local`? It would make sure that the nodes are kept local to each list.